framework/console/controllers/BaseMigrateController.php

```diff
@@ -895,4 +909,22 @@ protected function generateMigrationSourceCode($params)
      * @param string $version migration version name.
      */
     abstract protected function removeMigrationHistory($version);
+
+    /**
+     * @param ReversibleMigrationInterface|Migration $migration
+     */
+    protected function applyChangesFrom(ReversibleMigrationInterface $migration)
```

Comment:
the method naming could be more verbose. It would be more clear if the naming is about 'reversibility'. 
Perhabs something like:
*  `applyByReversibleMigration()` and the other one `revertByReversibleMigration`
* or `upByReversibleMigration()` and `downBy....`, which is even more consistent